presto-main/src/main/java/com/facebook/presto/operator/ExchangeClient.java

```diff
@@ -52,6 +52,11 @@
 import static io.airlift.slice.Slices.EMPTY_SLICE;
 import static java.util.Objects.requireNonNull;
 
+/**
+ * {@link ExchangeClient} is the client on receiver side, used in operators requires data exchange from other tasks,
+ * such as {@link ExchangeOperator} and {@link MergeOperator}.
+ * For each sender, a {@link HttpPageBufferClient} is used.
```

Comment:
`For each receiver` you mean?

Reply:
@highker : So the `ExchangeClient` can receive data from many other tasks. And a `HttpPageBufferClient` is used for each sending task.

What about 
```
For each sender the ExchangeClient receives data from, a {@link HttpPageBufferClient} is used in `ExchangeClient`.
```

Reply:
```
                                   /   HttpPageBufferClient_1  - - - Remote Source 1
                    ExchangeClient  - ...
                                    \  HttpPageBufferClient_n  - - - Remote Source n
```

Reply:
fair :D